styles.css

```diff
@@ -3,3 +3,46 @@
 *::after {
   box-sizing: border-box;
 }
+* {
+  font-family: "Maven Pro", sans-serif;
+}
+body {
+  background-color: #141414;
+  text-decoration: none;
+  margin: 0;
+}
+a {
+  text-decoration: none;
+}
+header {
+  height: 100px;
+  display: grid;
+  grid-template-columns: 3fr auto;
+  padding: 10px 0px 10px;
+}
+
+nav ul li {
+  display: inline-block;
+  font-weight: 600;
+  padding: 12px 12px;
+  color: #fff;
+  justify-content: space-between;
+  position: relative;
+  transition: 0.5s;
+  cursor: pointer;
+}
+nav ul li:hover {
+  color: #5b7678;
+}
+.logo {
+  display: grid;
```

Comment:
Just `display: grid` won't do much

Reply:
I started over to get a better hang on grids. Can you check again? THX